org.eclipse.debug.ui/ui/org/eclipse/debug/ui/actions/ContextualLaunchAction.java

```diff
@@ -237,15 +237,11 @@ protected void fillMenu(Menu menu) {
 					if (category != null && !categories.contains(category)) {
 						categories.add(category);
 					}
+					populateMenuItem(mode, ext, menu, null, accelerator++, null);
```

Comment:
how about passing empty string as the last parameter ?

Reply:
I fail to grok how this would be better?